src/actions/dashboardActions.ts

```diff
@@ -105,6 +113,23 @@ export function removeWidget(widgetId: String) {
   }
 }
 
+export function moveWidget(
+  sourceIndex: number,
+  sourceContainerId: string,
```

Comment:
Maybe just personal preference but 'sourceIndex' and 'sourceContainerId', little confused on what these are (the difference) maybe some comments here would help?

Reply:
Added comments